_includes/code/python/best-practices.python.ai.py

```diff
@@ -285,9 +285,8 @@
 Batch import for better performance
 """
 
-# Dynamic batch (adapts to Weaviate load)
-collection = client.collections.get("Article")
-with collection.batch.dynamic() as batch:
+# Fixed size batch (Recommended option)
+with collection.batch.fixed_size(batch_size=50) as batch:
```

Comment:
Heya, could we add this back in please `collection = client.collections.get("Article")`

Reply:
Great catch, my bad